### PR TITLE
chore[docs]: Tornado 5/6 support for Python 3.7+

### PR DESCRIPTION
**Pull request by Yun-Kim**
## Description
An error in the Tornado integration documentation was fixed to include Tornado 5/6 support for Python versions 3.7 and above.

### CODE REVIEW

File: ddtrace/contrib/tornado/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Auto instrumentation is available using the ``patch`` function that **must be called before**
 importing the tornado library.
 
-**Note:** Tornado 5 and 6 supported only for Python 3.7.
+**Note:** Tornado 5 and 6 supports Python 3.7 and above.
```
Comment (Kyle-Verhoog) [inline ddtrace/contrib/tornado/__init__.py:6]: Should we just get rid of this line completely? I don't really know what purpose it's serving

Comment (Yun-Kim) [reply ddtrace/contrib/tornado/__init__.py:6]: Interestingly enough, Tornado 5.X is compatible with Python 2.7 and 3.4+ while Tornado 6.X supports Python 3.5+. I don't see the point of this line unless our integration only supports specific versions of Python (will be testing out to see if we can indeed support 2.7 and 3.4-3.7).

Comment (Kyle-Verhoog) [reply ddtrace/contrib/tornado/__init__.py:6]: Yeah let's only call it out here if this is the case. Right now I think the message is confusing as it sounds like we're stating what versions tornado itself supports

Comment (majorgreys) [reply ddtrace/contrib/tornado/__init__.py:6]: The existing note is less confusing but not much better really. Perhaps a better alternative could be:

```
**Note:** This integration requires Python 3.7 and above for Tornado 5 and 6.
```

**Review by Kyle-Verhoog — APPROVED** [review]
:+1: good with this for now until we can confirm our support for other versions!

**Review by majorgreys — APPROVED** [review]

**Pull request merged**